spec/requests/api/reports_spec.rb

```diff
@@ -117,7 +161,7 @@
 
       expect do
         api_basic_authorize action_identifier(:reports, :run)
-        run_post "#{reports_url(report.id)}", :action => "run"
+        run_post reports_url(report.id).to_s, :action => "run"
```

Comment:
Sorry, probably wasn't very clear. There are two issues:
1. Neither interpolation nor `to_s` should be needed - just `reports_url(report.id)` (it should already be a string)
2. Unless this change is relevant to your enhancement, I would prefer not to change it here but in another PR

Thanks!